usr/discovery.c

```diff
@@ -823,6 +823,7 @@ iscsi_alloc_session(struct iscsi_sendtargets_config *config,
 	session->conn[0].noop_out_timeout = 0;
 	session->conn[0].noop_out_interval = 0;
 	session->reopen_cnt = config->reopen_max + 1;
+	session->reopen_log_freq = 1;
```

Comment:
It's strange that we have default value defines for these things but don't use them here ... But it's not your fault that the usr-vs-libopeniscsiusr database stuff is a mess of duplication. An contributor did that, and I should have caught it before merging his code.

Reply:
Changed to defaults.